packages/mini-browser/src/browser/mini-browser-open-handler.ts

```diff
@@ -159,7 +159,12 @@ export class MiniBrowserOpenHandler extends NavigatableWidgetOpenHandler<MiniBro
 
     protected resetBackground(uri: URI): MaybePromise<boolean> {
         const { scheme } = uri;
-        return scheme === 'http' || scheme === 'https' || (scheme === 'file' && uri.toString().endsWith('.html'));
+        const uriStr = uri.toString();
+        return scheme === 'http'
+            || scheme === 'https'
+            || (scheme === 'file'
+                && (uriStr.endsWith('html') || uriStr.endsWith('.xhtml') || uriStr.endsWith('.htm'))
```

Comment:
Nit: Strings ending with `.xhtml` also end with `html` already (not `.`).

As do `.mhtml` files (for which you also want to reset the mini-browser background, however no need to add `mhtml` in the other file because it doesn't make sense to open archives in Theia's editor).

```suggestion
                && (uriStr.endsWith('html') || uriStr.endsWith('.htm'))
```

Reply:
Updated :) Thank you for the review!